### PR TITLE
Removes prometheus actuator endpoints

### PR DESCRIPTION
**Pull request by yue9944882**

### CODE REVIEW

File: examples/examples-release-11/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,2 +1,3 @@
 namespace=airflow
-management.endpoints.web.exposure.include=prometheus
\ No newline at end of file
+management.endpoints.web.exposure.include=prometheus
+kubernetes.actuator.prometheus.enabled=true
```
Comment (shakuzen) [inline examples/examples-release-11/src/main/resources/application.properties:3]: I think this property is no longer used.

Comment (yue9944882) [reply examples/examples-release-11/src/main/resources/application.properties]: thanks, removed

**Comment by yue9944882** [comment]
@dsyer i updated to make this pull minimal/easiest. lgty?

**Pull request merged**